build/buildBuildImages.sh

```diff
@@ -361,28 +362,26 @@ function buildVsoFocalImage() {
 	echo "$builtImageName image history"
 	docker history $builtImageName
 
-	docker tag $builtImageName "$DEVBOX_BUILD_IMAGES_REPO:vso-focal"
-
-	echo
-	echo "$builtImageName" >> $ACR_BUILD_IMAGES_ARTIFACTS_FILE
+	docker tag $builtImageName "$DEVBOX_BUILD_IMAGES_REPO:ubuntu-vso-focal"
 }
 
 function buildCliImage() {
 	buildBuildScriptGeneratorImage
 	
 	local debianFlavor=$1
-	local devImageTag=cli
+	local devImageRepo="$DEVBOX_CLI_BUILD_IMAGE_REPO"
+	local devImageTag="debian-$debianFlavor"
 	local builtImageName="$ACR_CLI_BUILD_IMAGE_REPO"
 
-	if [ -z "$debianFlavor" ] || [ "$debianFlavor" == "stretch" ]; then
+	if [ -z "$debianFlavor" ] || [ $debianFlavor == "stretch" ] ; then
 		debianFlavor="stretch"
-	elif  [ "$debianFlavor" == "buster" ]; then
-		debianFlavor="buster"
-		devImageTag=$devImageTag-$debianFlavor
-		echo "dev image tag: "$devImageTag
-		builtImageName=$builtImageName-$debianFlavor
-		echo "built image name: "$builtImageName
+		builtImageName="$builtImageName:debian-$debianFlavor"
+	else
+		builtImageName="$builtImageName-$debianFlavor:debian-$debianFlavor"
+		devImageRepo="$DEVBOX_CLI_BUILD_IMAGE_REPO-buster"
```

Comment:
nit: separate from your work but would be nice to change the hard-coded `buster` here to `$debianFlavor` just to be safe for future flavor adoptions in the CLI image.